modules/labs-react/common/stories/examples/FlexItem.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import {Flex} from '@workday/canvas-kit-labs-react/layout';
+import {Box} from '@workday/canvas-kit-labs-react/common';
+
+export const FlexItem = () => (
+  <Flex padding="xs">
+    <Box flex={2} border="solid 1px" padding="xs" margin="xxxs">
```

Comment:
This should use `Flex.Item`, no?

Reply:
There's no `Flex.Item` component, as it would be identical to `Box`. `FlexItem` here refers to `flexItem` props:

- flex
- flexGrow
- flexShrink
- flexBasis
- justifySelf
- alignSelf
- order

Reply:
Oh right - sorry haha. My brain was stuck in our conversation from 2 weeks ago 😅